src/dbnode/persist/fs/commitlog/commit_log_test.go

```diff
@@ -130,6 +131,7 @@ func (w testWrite) assert(
 	unit xtime.Unit,
 	annotation []byte,
 ) {
+	fmt.Println("Doing this, series", int(w.series.UniqueIndex), "no w", int(series.UniqueIndex))
```

Comment:
delete

Reply:
Luckliy this is the println I left in rather than one of the more objectionable ones :p